recipe/meta.yaml

```diff
@@ -26,28 +26,28 @@ outputs:
       noarch: python
     requirements:
       host:
-        - pip
         - python >=3.8
-        - setuptools >=30.3.0
-        - setuptools_scm
+        - setuptools >=61
+        - setuptools-scm >=6.2
+        - pip
       run:
         - python >=3.8
+        - beautifulsoup4
+        - colorama
+        - conda-souschef >=2.2.3
+        - packaging >=21.3
+        - pip
+        - pkginfo
+        - progressbar2 >=3.53.0
+        - rapidfuzz >=1.7.1
         - requests
         - ruamel.yaml >=0.16.10
         - ruamel.yaml.jinja2
-        - stdlib-list
-        - pip
         - setuptools >=30.3.0
-        - rapidfuzz >=1.7.1
-        - progressbar2 >=3.53.0
-        - colorama
-        - conda-souschef >=2.2.3
-        - pkginfo
-        - packaging >=21.3
+        - semver >=2.13.0,<2.14.dev0
```

Comment:
@marcelotrevisani, I'm getting complaints from grayskull's internal pip check:
```
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
grayskull 2.3.0 requires semver~=2.13.0, but you have semver 3.0.0 which is incompatible.
```
This is because semver is underconstrained here on the feedstock.

Reply:
Indeed, I need to modify grayskull to also support version 3.0.0
thanks for fixing the recipe